### ISSUE TITLE
Checklist for getting to CRAN

### ISSUE THREAD

**Issue by andrie** [rstudio/sortable#15]
We're getting ready for a CRAN release, hopefully before the end of June.

House keeping:

- [x] Migrate repo to https://github.com/rstudio/sortable
- [x] Update Readme
- [x] Enable Travis
- [x] Add code coverage tests
- [x] Publish `pkgdown`
- [x] Review documentation and help for clarity
- [x] Add hex logo

Add vignettes:

- [x] Introduction to `sortable`
- [x] Using `sortable` in shiny apps
- [ ] New custom question types for `learnr` tutorials
- [ ] Creating your own custom `sortable` widgets
- [ ] Passing JavaScript functions to the underlying `sortable` JS library

Features:

- [x] Create parsons question type
- [x] Modify `sortable_list` to capture input immediately (not only after first sort)

Won't do:

- [ ] Use `shinytest` to test functionality of the JavaScript.  It seems that `shinytest` doesn't play well with `htmlwidgets` :-(

**Comment by andrie** [comment]
Closing because `sortable` was accepted on CRAN on 2019-11-01
